v03_pipeline/lib/misc/io.py

```diff
@@ -200,6 +202,17 @@ def import_pedigree(pedigree_path: str) -> hl.Table:
     )
 
 
+def remap_pedigree_hash(remap_path: str, pedigree_path: str) -> hl.Int32Expression:
+    sha256 = hashlib.sha256()
+    if hfs.exists(remap_path):
+        with hfs.open(remap_path) as f1:
+            sha256.update(f1.read().encode('utf8'))
+    with hfs.open(pedigree_path) as f2:
+        sha256.update(f2.read().encode('utf8'))
+    # maximum 4 byte int
+    return hl.int32(int(sha256.hexdigest()[:8], 16))
```

Comment:
What's the reasoning behind converting the hash value string to an int?

Reply:
Compression!  Ints are smaller.  We don't really need to use sha256 here since we're only taking 32 bits, but that's the func we're using elsewhere so I stuck with it.

Reply:
it's definitely a micro optimization though that will have very limited implications.  It wasn't hard though!